internal/monitor/controller.go

```diff
@@ -377,7 +411,7 @@ func (cm *PodMonitorType) callValidateVolumeHostConnectivity(node *v1.Node, volu
 				log.Info(message)
 			}
 		}
-		log.Infof("ValidateVolumeHostConnectivity Node %s NodeId %s Connected %t", node.ObjectMeta.Name, req.NodeId, resp.GetConnected())
+		log.Infof("ValidateVolumeHostConnectivity Node %s NodeId %s Connected %t IosInProgress %t", node.ObjectMeta.Name, req.NodeId, resp.GetConnected(), resp.GetIosInProgress())
```

Comment:
suggestion: same as above for punctuation.
```suggestion
		log.Infof("ValidateVolumeHostConnectivity Node: %s, NodeId: %s, Connected: %t, IosInProgress: %t", node.ObjectMeta.Name, req.NodeId, resp.GetConnected(), resp.GetIosInProgress())
```